src/encoded/schemas/atac_reproducibility_quality_metric.json

```diff
@@ -0,0 +1,110 @@
+{
+    "title": "ATAC-seq reproducibility quality metric",
+    "description": "Schema for reporting ATAC-seq reproducibility quality metrics",
+    "id": "/profiles/atac_reproducibility_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "reproducibility_type": {
```

Comment:
I think "reproducibility_type" should be required, otherwise we can end up with values without the ability to find out is it is IDR or not.